astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -14,39 +14,80 @@ This page contains notes on changes in recent versions of FusionAuth that requir
 
 ## Version 1.99.9
 
-Version `1.99.9` does not *require* any theme changes. There are new fields, for example `tenant.passwordEnabled`, that can be used to hide the password field on the hosted login page when this value is set to `true`.
+Version `1.99.9` does not *require* any theme changes. User passwords are optional in version `1.99.9` at the tenant level.
```

Comment:
If you like this, then I would suggest having 3 sections below, one for each of these, and re-working the content into them.
```suggestion
User passwords are optional starting in version `1.99.9`. While theme updates are not required, there are some areas that you may want to consider making changes:
* Advanced registration forms
* OAuth complete registration templates
* User edit forms (admin and self-service)
```